### PR TITLE
Add Pagination to tech-docs

### PR DESCRIPTION
**Pull request by imprashast**
This will add Pagination to our tech-docs. Since this is only React, had to do some tweaks to make it work

### CODE REVIEW

File: docs/components/pagination/Example.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+let paginationData = ref({
+    currentPage: 0,
+});
+onMounted(() => {
+  const root = createRoot(el.value)
+  root.render(createElement(Pagination, { currentPage: paginationData.value.currentPage,
+            numPages: 3,
+            lastPage: 3,
+            createHref: (page) => `?page=${page}`,
+            onChange: (page) => paginationData.value.currentPage++ }, null))
```
Comment (imprashast) [inline docs/components/pagination/Example.vue]: Does anyone know how to increment a counter in vue? @magnuh @felicia-haggqvist @BalbinaK

Comment (BalbinaK) [reply docs/components/pagination/Example.vue]: I tried to play around with it but got nowhere. Very curious what's the solution here 👀

Comment (felicia-haggqvist) [reply docs/components/pagination/Example.vue]: Yeah same here! 🙈 Are we sure that the onChange is being triggered in the react Pagination component?

Comment (BalbinaK) [reply docs/components/pagination/Example.vue]: I did manage to console log the correct value of `paginationData.value.currentPage` inside the `onChange`:
```
onChange: (page) => {
          paginationData.value.currentPage = page;
          console.log(`hello ${page} ${paginationData.value.currentPage}`); // hello 1 1 ... hello 2 2 ... hello 3 3
        }
```
 
 but the ref value doesn't seem to be updated outside. As though the Vue component didn't re-render.

**Review by felicia-haggqvist — APPROVED** [review]
🎉

**Review by BalbinaK — APPROVED** [review]
Should we publish @warp-ds/react v1.3.0 before merging? This way we make sure that  whatever is in the docs is also available in the packages. Wdyt?

**Pull request merged**